tox.ini

```diff
@@ -15,7 +15,8 @@ test_dir =
     ano: cli/anomaly
     cls: cli/classification
     det: cli/detection
-    seg: cli/segmentation
+    i_seg: cli/instance_segmentation
+    s_seg: cli/semantic_segmentation
```

Comment:
We don't need this separation. This cls/det/seg means just `extra` dependency in pip install.

Reply:
Done